src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1997,13 +2041,21 @@ export class StudyViewPageStore {
         geneFilters = _.reduce(
             geneFilters,
             (acc, next) => {
-                const newGroup = next.filter(oql => oql !== toBeRemoved);
+                const [
+                    hugoGeneSymbol,
+                    alteration,
+                ]: string[] = toBeRemoved.split(':');
```

Comment:
we super sure it will always have colon in it?

Reply:
No, we are not. But why would this matter?